B/bsdiff_classic/build_tarballs.jl

```diff
@@ -17,6 +17,7 @@ perl -i -ple '$_ = "#include <sys/types.h>\n" . $_ if $. == 31' bspatch.c
 cc -O3 -lbz2 -I"${prefix}/include" bsdiff.c -o bsdiff
 cc -O3 -lbz2 -I"${prefix}/include" bspatch.c -o bspatch
 install bsdiff bspatch "${bindir}"
+install_license bsdiff.c bspatch.c
```

Comment:
Using source code as license file isn't a great idea.  That seems to be 2-clause BSD, we have that somewhere in the rootfs

Reply:
It should be
```suggestion
install_license /usr/share/licenses/BSD2
```

Reply:
That file is missing the name and year

Reply:
I updated the script to add the top lines of the c file as a license with the correct name and year.

Reply:
I'm not particularly enthusiastic about cutting files around, what if in a next version the file slightly change (e.g. they add a new line at the top of the file)?  No one will tell you that the license file is now truncated.  We ship the license files precisely for these cases.

Reply:
Yes, but how should I replace the "Copyright (c) [year], [fullname]" in "/usr/share/licenses/BSD2" with "Copyright 2003-2005 Colin Percival"?